frontend/src/components/IstioWizards/Slider/slider.module.scss

```diff
@@ -0,0 +1,376 @@
+.style {
+  :global {
+    /* Extracted from Bootstrap CSS. Don't modify these styles */
```

Comment:
Should we indicate which version this was taken from?

Reply:
According to https://github.com/kiali/kiali-ui/pull/1673, these styles come from PF 3 where Bootstrap slider (v 9.9.0) and Bootstrap (v 3.4.1) libraries were used. I have compared style values with the CSS files that comes with the libraries and the values are a bit different (probably customized by PF3). I will update the comment.